### DISCUSSION TITLE
How to install etcd inside a EC2 and use it for key:value pair service for a external opensource tool

### DISCUSSION THREAD

**Discussion by RVaidhy** [etcd-io/etcd#17505] [Q&A]
Hi Team ,

We have configured a opensource tool (openobserve.ai) for logging inside an EC2. For a HA setup it requires etcd should be availlable even though we use postgres. Here, I have installed etcd in our EC2 but not be able to create user, set password and use. 

How to use etcd as a dedicated keypair service inside Ec2 ? Incase if we want to call the etcd inside for the below requirement means how to do that. Please help.

https://openobserve.ai/docs/ha_deployment/#postgresql

config:
  ZO_ETCD_PREFIX: "/zinc/observe/"
  ZO_ETCD_USER: ""
  ZO_ETCD_PASSWORD: ""
etcd:
  enabled: false # disable emebed etcd
  externalUrl: "my_custom_host.com:2379" # if bundled is false then this is required

Thanks,
Vaidhy R

**Comment by jmhbnz** [comment]
Hey @RVaidhy - Thanks for your question.  What have you tried for the etcd installation?

Please refer to: https://etcd.io/docs/v3.5/install in the first instance.
